app/models/contents_list.rb

```diff
@@ -0,0 +1,7 @@
+class ContentsList
```

Comment:
I think perhaps this should renamed as "ContentsList" implies a specific rendering choice, and this is really just a list of links. That should mean that it'll be more likely to be reused.

Reply:
This has been replaced with the existing ContentsOutline